README.md

```diff
@@ -1,17 +1,56 @@
-# linux-headers-musl
-
 [![Build Status](https://dev.azure.com/chefcorp-partnerengineering/Chef%20Base%20Plans/_apis/build/status/chef-base-plans.linux-headers-musl?branchName=master)](https://dev.azure.com/chefcorp-partnerengineering/Chef%20Base%20Plans/_build/latest?definitionId=170&branchName=master)
 
-Linux kernel headers (sanitized for use with musl)
+# linux-headers-musl
 
-## Maintainers
```

Comment:
The Maintainers header may have been accidentally removed.

<img width="676" alt="image" src="https://user-images.githubusercontent.com/16652757/85546555-ebce7a00-b614-11ea-92ff-4a7eacfa2c5f.png">